camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -1070,8 +1092,11 @@ def taradd(name, buf):
 
         for (name, im) in ims:
             taradd(name, cv2.imencode(".png", im)[1].tostring())
-
-        taradd('ost.txt', self.ost())
+	l_yaml, l_ost = self.ost("/left", self.l)
+	r_yaml, r_ost = self.ost("/right", self.r)
```

Comment:
do not use tabs in Python, use spaces, cf http://wiki.ros.org/PyStyleGuide#Coding_Style